projects/ui/src/lib/components/po-table/po-table-base.component.spec.ts

```diff
@@ -1267,5 +1267,13 @@ describe('PoTableBaseComponent:', () => {
 
       expect(component.nameColumnDetail).toBeNull();
     });
+
```

Comment:
Utilizar expectPropertiesValues em vez do expectSettersMethod, veja o p-loading como exemplo:

```
    it('p-loading: should update property `p-loading` with valid values.', () => {
      expectPropertiesValues(component, 'loading', booleanValidTrueValues, true);
    });

    it('p-loading: should update property `p-loading` with invalid values.', () => {
      expectPropertiesValues(component, 'loading', booleanInvalidValues, false);
    });
```

Criar um it para valores validos e um it para valores invalidos, conforme o exemplo acima.